blog-posts/1-meet-the-hoodie-@espy.md

```diff
@@ -0,0 +1,45 @@
+
+What I really like about it though is that it works, and it works well on my four-year-old Android phone. With photos, map tiles, the lot. It shows what was possible even with ”old” technology, such as AppCache and LocalStorage, and bodes well for all the brilliant new stuff that’s on the way.
+
+The best thing was that Hoodie saved me so much work implementing the authentication, data storage and sync that I could spend much more time on the UI and the user experience, which really paid off.
+
+###*What in your opinion makes the Hoodie project unique and interesting?*
+
+The people-centered approach to, well, basically everything. It starts with how we desigend the API: we didn‘t go down the tech-centered path you see so often in Open Source, simply exposing server-side features to the front-end, but focusing instead on what developers would need and want. User-centered API design.
+
+This approach extends to how we do work in general, trying to accomodate people and their needs as well as we can, and being open, inclusive and accessible, also valuing all sorts of work highly, not just coding. This interview, for example, is the product of our new Open Source editorial team, an attempt at finding contributors and contributions for Hoodie’s outward communication.
+
+We’ve often been told that people are attracted to how we do things as much as to the things that we do, and we feel very validated by this. Open Source is more than just code. Code changes and becomes deprecated, but culture and people persist far beyond that.
+
+I like that we value independence and autonomy highly, in that there's no VC funding, the project is completely Open Source, can’t be sold or acquired, and is generally a safe bet: it’s not going to vanish overnight, you’re not going to lose your data and your storage platform because we got bought, and we’re not going to drop the whole thing once a lucrative exit presents itself. Hoodie is our exit strategy. We want this to be A Thing™.
+
+###*What advice do you have for aspiring Hoodie contributors?*
+
+First off, don't be shy, we’re pretty easy to get in touch with, both over twitter @hoodiehq, as well as our combined Slack/IRC channel (find out more on our chat page). We’ve also had people just show up out of nowhere with a pull request on GitHub, that’s also cool. But in general, just come talk to us!
```

Comment:
Hm, I can edit (and therefor copy) yours, but I'm a repo owner, so that’s probably why. Here’s the [source markdown](https://gist.githubusercontent.com/espy/3155ac9c84065e0eceb6/raw/ad67ce5b703e2443f2993935d78a0bd68f313719/interview_alex.md).